include/tgfx/core/ColorFilter.h

```diff
@@ -65,6 +65,16 @@ class ColorFilter {
    */
   static std::shared_ptr<ColorFilter> Matrix(const std::array<float, 20>& rowMajor);
 
+  /**
+    * Creates a new ColorFilter that converts translucent colors to opaque colors
+    * based on the specified alpha threshold.
+    *
+    * Colors with alpha values equal to or below this threshold will be converted to fully
+    * transparent, while colors with alpha values above this threshold will be
+    * converted to fully opaque.
```

Comment:
Creates a new ColorFilter that makes translucent colors fully opaque or fully transparent based on a specified alpha threshold. Colors with alpha values at or below this threshold will become fully transparent, while colors with alpha values above this threshold will become fully opaque.